### ISSUE TITLE
[Crackle] Support downloading VTT subtitles

### ISSUE THREAD

**Issue by Mennaruuk** [yt-dlp/yt-dlp#1689]
### Checklist

- [X] I'm reporting a site feature request
- [X] I've verified that I'm running yt-dlp version **2021.11.10.1**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

United States

### Example URLs

https://www.crackle.com/watch/8890/2521052

### Description

Currently, yt-dlp cannot download subtitles from Crackle. It'd be handy to have it do so. Subtitles show up as a VTT header under the Network tab when choosing the subtitles manually. An example:

![image](https://user-images.githubusercontent.com/52135169/141921614-76365577-1dea-4978-8a22-4bb4548bf6bc.png)

The subtitles for this movie: https://www.crackle.com/watch/8890/2521052
are: https://widevinedashus-crackle-com.akamaized.net/1/3/cm/enc_ca3yb_en-US_211110222132.vtt

Verbose log:

```
yt-dlp https://www.crackle.com/watch/8890/2521052 --list-subs -Uv
[debug] Command-line config: ['https://www.crackle.com/watch/8890/2521052', '--list-subs', '-Uv']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, err utf-8, pref cp1252
[debug] yt-dlp version 2021.11.10.1 (win_exe)
[debug] Python version 3.8.10 (CPython 64bit) - Windows-10-10.0.19043-SP0
[debug] exe versions: none
[debug] Optional libraries: Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {}
yt-dlp is up to date (2021.11.10.1)
[debug] [Crackle] Extracting URL: https://www.crackle.com/watch/8890/2521052
WARNING: [Crackle] No country code was given using --geo-bypass-country. Trying with a list of known countries
[Crackle] 2521052: Downloading media JSON from US API
ERROR: [Crackle] 2521052: 2521052: This video is DRM protected
Traceback (most recent call last):
  File "yt_dlp\extractor\common.py", line 595, in extract
  File "yt_dlp\extractor\crackle.py", line 180, in _real_extract
  File "yt_dlp\extractor\common.py", line 1059, in report_drm
  File "yt_dlp\extractor\common.py", line 1100, in raise_no_formats
yt_dlp.utils.ExtractorError: 2521052: This video is DRM protected
Traceback (most recent call last):
  File "yt_dlp\extractor\common.py", line 595, in extract
  File "yt_dlp\extractor\crackle.py", line 180, in _real_extract
  File "yt_dlp\extractor\common.py", line 1059, in report_drm
  File "yt_dlp\extractor\common.py", line 1100, in raise_no_formats
yt_dlp.utils.ExtractorError: 2521052: This video is DRM protected

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "yt_dlp\YoutubeDL.py", line 1311, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1336, in __extract_info
  File "yt_dlp\extractor\common.py", line 621, in extract
yt_dlp.utils.ExtractorError: [Crackle] 2521052: 2521052: This video is DRM protected
```

**Comment by pukkandan** [comment]
> yt_dlp.utils.ExtractorError: [Crackle] 2521052: 2521052: This video is DRM protected

Please provide non-DRM example

**Comment by Mennaruuk** [comment]
All Crackle videos are DRM protected, so IMO Crackle's being in the supported sites list should be revisited. But I'm only trying to download the subtitles, not the videos.

**Comment by pukkandan** [comment]
```
➤ yt-dlp https://www.crackle.com/watch/8890/2521052 --list-subs --ignore-no-formats
WARNING: [Crackle] No country code was given using --geo-bypass-country. Trying with a list of known countries
[Crackle] 2521052: Downloading media JSON from US API
WARNING: [Crackle] 2521052: This video is DRM protected
WARNING: No video formats found!
Available subtitles for 2521052:
Language Formats
en-US    vtt, xml
```

> All Crackle videos are DRM protected

I'm pretty sure it wasn't the case when the extractor was written/last updated.

You can extract the subs in this case because the code for it happens to be the same as when the video doesn't have DRM. Any requests for enhancement/bug-fixes wont be accepted unless non-DRM examples can be provided - even if you are not trying to download the video itself
